expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +240,20 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
+                step_dir = 1
+                self._n_up = 0
+                reversal = True
+                self._n_reversals += 1
+                self._direction = 1
+            if self._x[-1] == self._x_max:
+                step_dir = -1
+                self._n_up = 0
+                reversal = True
+                self._n_reversals += 1
+                self._direction = 1
+
```

Comment:
I'd say warn when the tracker stops.

Also it might only matter if the last `N` reversals were at floor/ceiling (e.g., usually those at the last step size). So it would be good to expose some property that the experimenter can check like `tr.check_valid(5)` would check to see if the last 5 reversals were valid (non-ceiling, non-floor). WDYT?